apps/mysql.go

```diff
@@ -57,9 +57,7 @@ func (r MetricsReceiverMySql) Pipelines() []otel.ReceiverPipeline {
 		r.Username = "root"
 	}
 
-	// MySQL replication metrics are implemented separate to the main metrics pipeline so that 5.7 and 8.0 are both supported,
```

Comment:
Sorry to be unclear, I was referring to the second part of this comment ("this configuration is used for…") earlier. This part was still useful.